simulations/vip-190/vip-190-testnet/simulations.ts

```diff
@@ -0,0 +1,87 @@
+      resilientOracle = new ethers.Contract(RESILIENT_ORACLE, RESILIENT_ORACLE_ABI, provider);
+      redStoneOracle = new ethers.Contract(REDSTONE_ORACLE, CHAINLINK_ORACLE_ABI, provider);
+      boundValidator = new ethers.Contract(BOUND_VALIDATOR, BOUND_VALIDATOR_ABI, provider);
+    });
+
+    it("Validate pivot oracle is not set for TRX_OLD", async () => {
+      const tokenConfig = await resilientOracle.getTokenConfig(TRX_OLD);
+      expect(tokenConfig.oracles[1]).to.equal(ethers.constants.AddressZero);
+      expect(tokenConfig.enableFlagsForOracles[1]).to.equal(false);
+    });
+    it("Validate pivot oracle is not set for TRX", async () => {
+      const tokenConfig = await resilientOracle.getTokenConfig(TRX);
+      expect(tokenConfig.oracles[1]).to.equal(ethers.constants.AddressZero);
+      expect(tokenConfig.enableFlagsForOracles[1]).to.equal(false);
+    });
+    it("Pending owner of RedStone oracle is Normal Timelock ", async () => {
+      const pendingOwner = await redStoneOracle.pendingOwner();
+      expect(pendingOwner).to.equal(NORMAL_TIMELOCK);
+    });
+  });
+
+  testVip("VIP-123 Change Oracle and Configure Resilient Oracle", vip190Testnet(), {
+    callbackAfterExecution: async txResponse => {
+      await expectEvents(txResponse, [CHAINLINK_ORACLE_ABI], ["TokenConfigAdded"], [2]);
+      await expectEvents(txResponse, [BOUND_VALIDATOR_ABI], ["ValidateConfigAdded"], [2]);
+      await expectEvents(txResponse, [RESILIENT_ORACLE_ABI], ["OracleSet"], [2]);
+      await expectEvents(txResponse, [RESILIENT_ORACLE_ABI], ["OracleEnabled"], [2]);
+    },
+  });
+
+  describe("Post-VIP behavior", async () => {
+    it("Owner of RedStone oracle is Normal Timelock ", async () => {
+      const owner = await redStoneOracle.owner();
+      expect(owner).to.equal(NORMAL_TIMELOCK);
+    });
+    it("Validate pivot oracle is set for TRX_OLD", async () => {
+      const tokenConfig = await resilientOracle.getTokenConfig(TRX_OLD);
+      expect(tokenConfig.oracles[1]).to.equal(REDSTONE_ORACLE);
+      expect(tokenConfig.enableFlagsForOracles[1]).to.equal(true);
+    });
+    it("Validate pivot oracle is set for TRX", async () => {
+      const tokenConfig = await resilientOracle.getTokenConfig(TRX);
+      expect(tokenConfig.oracles[1]).to.equal(REDSTONE_ORACLE);
+      expect(tokenConfig.enableFlagsForOracles[1]).to.equal(true);
+    });
+    it("Validate upper and lower bounds for TRX", async () => {
+      const validateConfig = await boundValidator.validateConfigs(TRX);
+      expect(validateConfig.upperBoundRatio).to.equal(PRICE_UPPER_BOUND);
+      expect(validateConfig.lowerBoundRatio).to.equal(PRICE_LOWER_BOUND);
+    });
+    it("Validate upper and lower bounds for TRX_OLD", async () => {
+      const validateConfig = await boundValidator.validateConfigs(TRX_OLD);
+      expect(validateConfig.upperBoundRatio).to.equal(PRICE_UPPER_BOUND);
+      expect(validateConfig.lowerBoundRatio).to.equal(PRICE_LOWER_BOUND);
+    });
+  });
```

Comment:
I would add a test case getting the price for TRX and TRX_OLD, validating it works with the new setup. I suppose we should invoke `setMaxStalePeriodInChainlinkOracle` two times (one for the Chainlink oracle and another one for the RedStone oracle)